controllers/weather.controller.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+import { requestOptions } from "../utils/requestOptions.js";
+
+export class WeatherController {
+  static async getWeather(req, res) {
+    await axios
+      .request(requestOptions(req.query.cityName))
+      .then((response) => {
+        if (response.status === 200) {
+          res.status(200).json(response.data);
+        } else {
+          res.status(response.status).json(response.data);
```

Comment:
what the difference between if and else? Both of them sends `res.status(response.status).json(response.data);`